### PR TITLE
Portfolio details popup window

### PR DESCRIPTION
**Pull request by LethaboMot**
In the project we:

-Added one more Javascript file to the project
-Added popups for every card in the project
-Added popup for mobile and desktop versions

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -93,9 +93,50 @@ <h2 class="heading">Tonic</h2>
...
+                                    <li>Javascript</li>
+                                </ul>
+
+                                <div class="buttons">
+                                    <div class="alive-button">
+                                        <button type="button" class="bluebutton">See live <img src="./images/golive.svg"/></button>
+                                        <button type="button" class="bluebutton">See source  <img src="./images/bluegithub.svg"></button>
+                                    </div>
+                                </div>
+
```
Comment (jerryowusu) [inline index.html:129]: As part of the project requirements, kindly note that the details popup window is supposed to be created dynamically by JavaScript and not inside the HTML. :confused:  So consider removing this and all subsequent ones and create it dynamically. :pray: 

After creating it dynamically, kindly ensure that: 

- [ ] When the user clicks (or taps) the button to check project details, the popup with details about the project appears, it should also dissappear when the user clicks (or taps) the close (X) button. :wink: 

- [ ] Kindly make the popup responsive, it fits mobile and desktop sizes properly as well as follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, text, margins). :pray:

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @LethaboMot , 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

almost there2

## Highlights
All linters are passing :heavy_check_mark: 
Great effort so far :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (LethaboMot) [reply index.html:129]: I don't understand, I used the javascript in a js file. what you are showing me is the HTML and for how the popup is supposed to be layout

File: index.html [hunk 2]
```diff
@@ -93,9 +93,50 @@ <h2 class="heading">Tonic</h2>
...
+                                    <li>Css</li>
+                                    <li>Javascript</li>
+                                </ul>
+
+                                <div class="buttons">
+                                    <div class="alive-button">
+                                        <button type="button" class="bluebutton">See live <img src="./images/golive.svg"/></button>
+                                        <button type="button" class="bluebutton">See source  <img src="./images/bluegithub.svg"></button>
+                                    </div>
+                                </div>
```
Comment (MaganaAsiati) [inline index.html:128]: -  kindly make sure that the details popup window are created dynamically by JavaScript and not inside the HTML as required [here](https://github.com/LethaboMot/Port-folio/pull/13/files#r1055403262) by the previous reviewer. Please remove this part and all subsequent ones and create it dynamically and apply it for all cards :+1: :sunglasses: :sunglasses: :sunglasses:
Here bellow a small explanation of how to do it :+1: 

- The JavaScript code implements an array of objects to store JavaScript objects containing the information for each project.
- Each object contains at least the following information about the project: name, description, featured image, technologies, link to live version, and link to the project source code. Ie:

```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 

Hi @LethaboMot 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlonzoLeroneThisCloseGIF

## Highlights
✅ No linter error
✅ README is professional

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @LethaboMot and @shasherazi  :wave: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well trying to work on the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there.



 ## Highlights :+1: 
:heavy_check_mark: Good commit history.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Your project is well documented


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**